packages/core/src/api/authenticate.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Let's check whether we need the `All Rights Reserved.` in the license header

Reply:
Used a newer version of eslint and prettier packages and corrected this. (b236cc1a3f74c07b1318260786ce3c4fbcf4f923)